thruster_control/include/thruster_control/CANIntf.h

```diff
@@ -93,13 +93,13 @@ class CANIntf
         bool SetupCANBus();
         bool isInitialized;
 
-    	bool DMCHeartbeatTimeout;
-		double motorTimeoutSeconds;
+        bool DMCHeartbeatTimeout;
+        double motorTimeoutSeconds;
 
-		bool enableCANBusLogging;
+        bool enableCANBusLogging;
 
-		std::vector<std::string> canNodeIdList;
+        std::vector<std::string> canNodeIdList;
 
 };
 
-#endif // _DATA_OBJECT_H
+#endif // THRUSTER_CONTROL_DATA_OBJECT_H

```

Comment:
@patapufate did you mean
```suggestion
#endif // THRUSTER_CONTROL_CAN_INTERFACE_H
```
?

Reply:
Fixed